packages/web-components/src/components/slider/slider.ts

```diff
@@ -569,7 +577,9 @@ class CDSSlider extends HostListenerMixin(FormMixin(FocusMixin(LitElement))) {
 
     return html`
       <label class="${labelClasses}" @click="${handleClickLabel}">
-        <slot name="label-text">${labelText}</slot>
+        <slot name="label-text">
+          ${labelText && !hideLabel ? labelText : null}
+        </slot>
```

Comment:
I think following what React does, we want to visually hide the label but still render out the label in the DOM so I would probably leave it as is and just have the `${prefix}--visually-hidden` class applied